### PR TITLE
Add landing page

### LINKED ISSUE #840: Landing page
Provide users a landing page to help understand what's in the system. Lou has a mock here:
https://projects.invisionapp.com/d/main#/console/13746874/329583851/preview

"Browse data" should link to /#browse-data
"Explore tools" should link to https://dockstore.org/search?descriptorType=wdl&searchMode=files 
"Analyze & Publish" should linke to /#workspaces

This page should be outside our logged-in boundary (as is dockstore and browse-data) but still behind the whitelist.

### PR DESCRIPTION
**Pull request by zarsky-broad**
Closes #840

### CODE REVIEW

File: src/components/FooterWrapper.js [hunk 1]
```diff
@@ -11,7 +12,7 @@ const styles = {
 const tosUrl = 'http://gatkforums.broadinstitute.org/firecloud/discussion/6819/firecloud-terms-of-service#latest'
 
 const FooterWrapper = ({ children }) => {
-  return div({ style: { display: 'flex', flexDirection: 'column', minHeight: '100%' } }, [
+  return div({ style: { display: 'flex', flexDirection: 'column', minHeight: '100%', flexGrow: 1 } }, [
```
Comment (panentheos) [inline src/components/FooterWrapper.js:15]: nit: this is unnecessary since the parent is not a flex

**Review by panentheos — APPROVED** [review]
Looks very nice. The hero image is huge, though. We should send it through tinypng.

File: src/components/FooterWrapper.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/FooterWrapper.js:15]: Without it, the 404 page and this one don't fill the screen beyond the height of their content.

**Comment by zarsky-broad** [comment]
TIL about tinypng!

File: src/pages/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        backgroundRepeat: 'no-repeat', backgroundSize: '750px', backgroundPosition: 'right -75px top -110px'
+      }
+    }, [
+      img({ src: landingPageHero, style: { position: 'absolute', right: 0, top: 60 } }),
+      div({ style: { fontSize: 22 } }, 'Welcome to'),
+      div({ style: { fontSize: 85 } }, 'Terra'),
+      hexLink('Access data from a rich ecosystem\nof Terra-connected data portals', 'Browse\nData', Nav.getLink('browse-data')),
+      div({ style: { margin: '-0.5rem 0 -0.5rem 3rem' } }, [
+        hexLink('Find ready-for-use bioinformatics workflows\nfrom the Dockstore workflow repository', 'Explore\nTools',
+          'https://dockstore.org/search?descriptorType=wdl&searchMode=files')
```
Comment (katevoss) [inline src/pages/LandingPage.js:52]: Should this open in a new tab, since it is off of Saturn?

**Review by katevoss — APPROVED** [review]

File: src/pages/LandingPage.js (hunk 2 above)
Comment (elizzhao) [reply src/pages/LandingPage.js:52]: Also wondered this!

**Review by elizzhao — APPROVED** [review]
Looks awesome!!

**Pull request merged**